resources/components/ImageCard.vue

```diff
@@ -19,17 +19,39 @@
 				</ul>
 			</div>
 
-			<div class="wbmad-buttons">
-				BUTTONS GO HERE
+			<div class="wbmad-action-buttons">
+				<base-button
+					class="wbmad-action-buttons__publish"
+					v-bind:primary="true"
+					v-bind:progressive="true"
+					v-bind:title="$i18n( 'machinevision-publish-title' )"
+					v-on:click="onPublish"
+				>
+					<span v-i18n-html:machinevision-publish />
+				</base-button>
+				<base-button
+					class="wbmad-action-buttons__skip"
+					v-bind:framed="false"
+					v-bind:title="$i18n( 'machinevision-skip-title', title ).parse()"
+					v-on:click="onSkip"
+				>
+					<span v-i18n-html:machinevision-skip />
+				</base-button>
 			</div>
 		</div>
 	</div>
 </template>
 
 <script>
+var Button = require( './base/Button.vue' );
+
 module.exports = {
 	name: 'ImageCard',
 
+	components: {
+		'base-button': Button
```

Comment:
I hit this issue too in my repo (that Button is reserved for button tag)

I was not sure what to do here - I called it btn and i guess base-button was another workaround attempt?
I'd love to use the same name as you in my repo.

Maybe wmf-button ?

Reply:
I've been thinking about prefixes for this and class names...I'd like to keep it as short and simple as possible, so I would recommend either `mw-` or `w-`. But I have a feeling this decision is going to be a Whole Thing 😅

Reply:
I'm happy with using mw-button. No need for this to be a thing :)